CHANGELOG.md

```diff
@@ -42,10 +42,14 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [\#104](https://github.com/cosmos/gaia/issues/104) Fix `ModuleAccountAddrs` to
 not rely on the `x/supply` keeper to get module account addresses for blacklisting.
 
-### Breaking Changes
+### State Machine Breaking Changes
 
 * (sdk) Update SDK version to v0.37.0
```

Comment:
Not sure where this line is supposed to go. May belong in multiple (if not all) subsections